README.md

```diff
@@ -96,38 +95,61 @@ To set properties from MQTT messages, add a setter function when advertising the
 ```
 var HomieDevice = require('homie-device');
 var myDevice = new HomieDevice('my-device');
-var myNode = myDevice.node('my-node', 'test-node');
-myNode.advertise('my-property-1').settable(function(range, value) {
+var myNode = myDevice.node('my-node', 'test node friendly name', 'test-node');
```

Comment:
This change was implemented as part of the original PR, so I just updated the readme to match.  Based on [this comment](https://github.com/microclimates/homie-device/pull/8#discussion_r247276250), it looks like this was done to match the [homie-esp8266 API](https://github.com/homieiot/homie-esp8266/blob/develop-v3/src/HomieNode.hpp#L81) for v3.

I think it could go either way (id, type, name vs id, name, type), but also keep in mind that friendly name is now required in v3, so it would still be a breaking change either way.

Reply:
Good call. Long term it's better to match homie-esp8266 than to retain backward compatibility - it's a major rev. change so this may be the only action people need to take when updating.